Components/Text Utilities/text utils.css

```diff
@@ -0,0 +1,17 @@
+@import "../../Colors/colors.css";
+
+.heading {
+  font-size: 3rem;
+}
+
+.small-text {
+  font-size: 1.5rem;
+}
+
+.gray-area {
+  color: var(--grey-color);
+}
```

Comment:
``` suggestion
Is this for text? if yes then you can name it gray or gray-text.
````

Reply:
Yes, will surely do. 
Thanks for reviewing Rohan